plugins/aiops/pkg/gateway/metricai.go

```diff
@@ -0,0 +1,363 @@
+package gateway
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	grafanav1alpha1 "github.com/grafana-operator/grafana-operator/v4/api/integreatly/v1alpha1"
+	"github.com/nats-io/nats.go"
+	"github.com/rancher/opni/pkg/resources"
+	metricai "github.com/rancher/opni/plugins/aiops/pkg/apis/metricai"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const serverUrl = "http://metric-ai-service:8090" // URL of the python metric-ai service
+const jobRunDelimiter = "="                       // delimiter that splits jobid and suffix, in order to save in natsKV
+const dashboardNamePrefix = "metricai-"
+
```

Comment:
Stylistically I prefer to group these up, i.e
```go
const (
	serverUrl = "http://metric-ai-service:8090" // URL of the python metric-ai service
	jobRunDelimiter = "="                       // delimiter that splits jobid and suffix, in order to save in natsKV
	dashboardNamePrefix = "metricai-"
)
```

Reply:
updated